### DISCUSSION TITLE
How to uninstall Betterfox?

### DISCUSSION THREAD

**Discussion by bdr99** [yokoffing/Betterfox#224] [Q&A] [answered]
I'm using Firefox 117.0 on Windows.

I installed Betterfox using the instructions in the readme, but now I want to uninstall it to revert the customizations made in user.js. I tried deleting user.js and restarting Firefox, but the customizations are still present.

How can I uninstall Betterfox?

**Comment by yokoffing** [comment, marked as answer]
> I want to uninstall it to revert the customization

Is there a reason? Is it something the [overrides](https://github.com/yokoffing/Betterfox/wiki/Common-Overrides) does not address?

> How can I uninstall Betterfox?

1) There are two native solutions: https://support.mozilla.org/en-US/kb/reset-preferences-fix-problems
2) or, Manually move your bookmarks, etc. to a new Firefox profile (see [here](https://github.com/yokoffing/Betterfox/issues/180#issuecomment-1622113788))
3) Otherwise, change the prefs back manually.

Soon, we will update the wiki to warn users to backup their profile (forthcoming).

**Reply by bdr99** [reply]
Thanks for your reply!

> Is there a reason? Is it something the [overrides](https://github.com/yokoffing/Betterfox/wiki/Common-Overrides) does not address?

I initially installed Betterfox with all of the configs (Fastfox, Securefox, etc) but later decided I wanted only certain configs.

> Soon, we will update the wiki to warn users to backup their profile (forthcoming).

That would be great. The readme didn't make clear at all that these changes would permanently alter my Firefox profile. I figured that since all of the changes were stored in user.js, deleting user.js would revert the changes.

**Reply by yokoffing** [reply]
> The readme didn't make clear at all that these changes would permanently alter my Firefox profile.

It's hinted at in places, but it could be more overt. In recent months, there's been a influx of new users who have no experience with user.js files, so I agree that it's needed. This documentation will be in place in the near future.

**Reply by yokoffing** [reply]
Created a placeholder and added to README: https://github.com/yokoffing/Betterfox/wiki/Backup
